### PR TITLE
👌Improve developer workflow

### PR DESCRIPTION
**Pull request by yngvark**
## Description

When developing #9, I found lots of stuff to improve for the general development of upgrades.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,117 @@ changes to reuse logic doesn't break any of the migrations using the common logi
...
+
+### Test continuously while developing
+
+You need two things for running the upgrade directly (i.e. not using `okctl upgrade`).
+
+* Environment variables
+
+Every upgrade must run with the same environment as `okctl show credentials` (which equals `okctl venv`).
+
+So if running through an IDE such as IntelliJ, add environmental variables to be at least those from `okctl show credentials`.
```
Comment (deifyed) [inline README.md:55]: This could probably benefit from `okctl show credentials > .env.local` and a description on how to use the env file in intellij

Comment (deifyed) [reply README.md:55]: Might have to ensure only stdout or stderr for that one. Can't remember which one printed only the commands

**Review by deifyed — APPROVED** [review]
Bash script knowledge level: 9001

**Review by haavardeide — APPROVED** [review]
:shipit: 
The step-by-step looks great without even having done this, so looking forward to test this in practice!

File: README.md [hunk 2]
```diff
@@ -50,32 +155,22 @@ git push --atomic origin main $TAG
...
-* delete the release in GitHub
-* delete tag
+* delete release and the tag corresponding tag in GitHub
+* update the upgrade code, and create a new release as described by the steps above.
 
-```shell
-git tag -d $TAG
-git push --delete origin $TAG
-```
+Note: Update existing released upgrades with care, as some users may have already downloaded and executed them. Also note that upgrades binaries are downloaded and cached when running okctl upgrade.
```
Comment (yngvark) [inline README.md:165]: Caching is disabled in https://github.com/oslokommune/okctl/pull/888:

```suggestion
Note: Update existing released upgrades with care, as some users may have already executed them.
```

**Pull request merged**